### PR TITLE
Cyclocity is the operator of bike rental schemes

### PR DESCRIPTION
**Pull request by gileri**
[While JCDecaux owns Cyclocity, it is Cyclocity which operates this activity](https://fr.wikipedia.org/wiki/Cyclocity).

### CODE REVIEW

**Comment by arch0345** [comment]
If this is the case, then the `operator:wikidata` tags should be changed to [Q3008464](https://wikidata.org/wiki/Q3008464)

File: data/brands/amenity/bicycle_rental.json [hunk 1]
```diff
@@ -2052,6 +2052,7 @@
         "brand": "Le vélo",
         "name": "Le vélo",
         "operator": "Cyclocity"
```
Comment (arch0345) [inline data/brands/amenity/bicycle_rental.json:2054]:
```suggestion
        "operator": "Cyclocity",
```

File: data/brands/amenity/bicycle_rental.json [hunk 2]
```diff
@@ -3877,6 +3878,7 @@
         "network": "VélO²",
         "network:wikidata": "Q3564030",
         "operator": "Cyclocity"
```
Comment (arch0345) [inline data/brands/amenity/bicycle_rental.json:3880]:
```suggestion
        "operator": "Cyclocity",
```

**Comment by gileri** [comment]
Ty for the linting fixes, my bad.

**Pull request merged**